.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -1350,6 +1350,7 @@ writingjavadocchecks
 writinglisteners
 Wsdl
 www
+xa
```

Comment:
This looks like it is from ` &#xa0;` which is the same as `&nbsp;`. We should use `&nbsp;` since it is already whitelisted.

Reply:
@Vampire ,
I am not sure what for you try to use https://en.wikipedia.org/wiki/Non-breaking_space after `\` in command lines.
On small screens commands are look not good, but if we want to make them better  it is better to use such symbol in even before `\`.
Please move this update to separate issue/PR. I doubt it is related to plugin validation, correct me if I am wrong.

Reply:
They are not for nicer look, but to get indentation with the `pre-line` style which strips leading whitespace.
For getting the markup valid I also had to change the styles a bit and tweaked it alongside as described in my other comment

Reply:
My original complaint is not valid. I will leave @romani to discuss this indentation usage.
I will mark this resolved.